.sync.yml

```diff
@@ -5,9 +5,15 @@
     - set: docker/ubuntu-14.04
   docker_defaults:
     bundler_args: ""
+    script: bundle exec rake spec_prep beaker
```

Comment:
I think you want to change this to `task_acceptance` as well, so next run of pdk won't revert the travis changes.